crypto/cipher/aes_icm.c

```diff
@@ -296,10 +296,10 @@ static void srtp_aes_icm_advance(srtp_aes_icm_ctx_t *c)
 
 static srtp_err_status_t srtp_aes_icm_encrypt(void *cv,
                                               unsigned char *buf,
-                                              unsigned int *enc_len)
+                                              size_t *enc_len)
 {
     srtp_aes_icm_ctx_t *c = (srtp_aes_icm_ctx_t *)cv;
-    unsigned int bytes_to_encr = *enc_len;
+    unsigned int bytes_to_encr = (unsigned int)*enc_len;
```

Comment:
Why is this one not also `size_t`?

Reply:
Because changing it led to a lot of other changes in in the aes code that I think require more understanding. Should maybe do it as it's own commit.